### PR TITLE
language_pl.h

### PR DESCRIPTION
**Pull request by ghost**
Fixed translations.

### CODE REVIEW

File: Marlin/Version.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
   /**
    * Marlin release version identifier
    */
-  #define SHORT_BUILD_VERSION "1.1.0-RCBugFix"
+  #define SHORT_BUILD_VERSION "1.1.0-RC7"
```
Comment (thinkyhead) [inline Marlin/Version.h:38]: Don't alter this file.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
 # Marlin 3D Printer Firmware
 
-[![Build Status](https://travis-ci.org/MarlinFirmware/Marlin.svg?branch=RCBugFix)](https://travis-ci.org/MarlinFirmware/Marlin)
-[![Coverity Scan Build Status](https://scan.coverity.com/projects/2224/badge.svg)](https://scan.coverity.com/projects/2224)
-
```
Comment (thinkyhead) [inline README.md:5]: Don't alter this file.

**Review by thinkyhead — CHANGES_REQUESTED** [review]
Thanks for the update! I've requested some changes. Please apply them and this can be merged.

File: README.md (hunk 2 above)
Comment (c64pl) [reply README.md:5]: _Please apply them and this can be merged_

Where?

Comment (thinkyhead) [reply README.md:5]: Fix the things I pointed out, commit the changes to your branch, and push them to "origin" to update the PR. If you're not sure how to manage a PR please read the Github documentation.

**Review by ghost — COMMENTED** [review]
Its ok.

File: README.md (hunk 2 above)
Comment (c64pl) [reply README.md:5]: I can not merge. There is no "merge pull request"

**Review by ghost — COMMENTED** [review]
aaa

**Comment by thinkyhead** [comment]
@c64pl Are you still unable to figure out how to update your branch? Github has some good info on how to do this. And Google is also a great resource.

**Comment by thinkyhead** [comment]
@c64pl I will redo this PR and submit it shortly.

Thanks for your submission! The following strings are still missing. I will insert placeholders in English into the Polish language file to give you a starting-point for new translations.
```
MSG_AUTO_HOME_X
MSG_AUTO_HOME_Y
MSG_AUTO_HOME_Z
MSG_BLTOUCH_RESET
MSG_BLTOUCH_SELFTEST
MSG_DAC_EEPROM_WRITE
MSG_DAC_PERCENT
MSG_DRIVE_STRENGTH
MSG_ERR_Z_HOMING
MSG_FILAMENT_CHANGE_EXTRUDE_1
MSG_FILAMENT_CHANGE_EXTRUDE_2
MSG_FILAMENT_CHANGE_EXTRUDE_3
MSG_FILAMENT_CHANGE_HEADER
MSG_FILAMENT_CHANGE_INIT_1
MSG_FILAMENT_CHANGE_INIT_2
MSG_FILAMENT_CHANGE_INIT_3
MSG_FILAMENT_CHANGE_INSERT_1
MSG_FILAMENT_CHANGE_INSERT_2
MSG_FILAMENT_CHANGE_INSERT_3
MSG_FILAMENT_CHANGE_LOAD_1
MSG_FILAMENT_CHANGE_LOAD_2
MSG_FILAMENT_CHANGE_LOAD_3
MSG_FILAMENT_CHANGE_OPTION_EXTRUDE
MSG_FILAMENT_CHANGE_OPTION_HEADER
MSG_FILAMENT_CHANGE_OPTION_RESUME
MSG_FILAMENT_CHANGE_RESUME_1
MSG_FILAMENT_CHANGE_RESUME_2
MSG_FILAMENT_CHANGE_RESUME_3
MSG_FILAMENT_CHANGE_UNLOAD_1
MSG_FILAMENT_CHANGE_UNLOAD_2
MSG_FILAMENT_CHANGE_UNLOAD_3
MSG_HALTED
MSG_INFO_BAUDRATE
MSG_INFO_BOARD_MENU
MSG_INFO_COMPLETED_PRINTS
MSG_INFO_EXTRUDERS
MSG_INFO_MAX_TEMP
MSG_INFO_MENU
MSG_INFO_MIN_TEMP
MSG_INFO_PRINTER_MENU
MSG_INFO_PRINT_COUNT
MSG_INFO_PRINT_FILAMENT
MSG_INFO_PRINT_LONGEST
MSG_INFO_PRINT_TIME
MSG_INFO_PROTOCOL
MSG_INFO_PSU
MSG_INFO_STATS_MENU
MSG_INFO_THERMISTOR_MENU
MSG_LIGHTS_OFF
MSG_LIGHTS_ON
MSG_PLEASE_RESET
MSG_SELECT
MSG_SHORT_DAY
MSG_SHORT_HOUR
MSG_SHORT_MINUTE
```

**Pull request closed without merging**